addons/hr_expense/wizard/hr_expense_sheet_register_payment.py

```diff
@@ -102,6 +102,13 @@ def _get_payment_vals(self):
             'communication': self.communication
         }
 
+    def _prepare_lines_to_reconcile(self, lines):
+        account_move_lines_to_reconcile = self.env['account.move.line']
+        for line in lines:
+            if line.account_id.internal_type == 'payable' and not line.reconciled:
+                account_move_lines_to_reconcile |= line
+        return account_move_lines_to_reconcile
```

Comment:
This is easier to understand
```suggestion
        return lines.filtered(lambda l: l.account_id.internal_type == 'payable' and not l.reconciled)
```

Reply:
> This is easier to understand
> ```suggestion
>         return lines.filtered(lambda l: l.account_id.internal_type == 'payable' and not l.reconciled)
> ```

Yes sir